php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -546,7 +550,30 @@ public UseStatementTree useStatement(
     InternalSyntaxToken eosToken
   ) {
     SeparatedListImpl<UseClauseTree> declarations = separatedList(firstDeclaration, additionalDeclarations);
-    return new UseStatementTreeImpl(useToken, useTypeToken.orNull(), declarations, eosToken);
+    return UseStatementTreeImpl.createUseStatement(useToken, useTypeToken.orNull(), declarations, eosToken);
+  }
+
+  public UseStatementTree groupUseStatement(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 9 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00107)